packages/accordion/accordion.scss

```diff
@@ -34,6 +34,7 @@
         padding: $title-padding;
         text-align: left;
         width: 100%;
+        display: flex;
```

Comment:
Is this for distributing content inside the title? In that case it should probably be `display: flex` on the `.jkl-accordion__title-text` class instead (which would then not need `width: inherit`).